package.json

```diff
@@ -90,6 +90,7 @@
   },
   "devDependencies": {
     "@babel/core": "7.9.6",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
@adamgibs I thought we figured this issue out?

Reply:
Not that I'm aware of. This is the only solution that worked for Christian and I.